repose-aggregator/tests/release-verification/src/vagrant/rpm/Vagrantfile

```diff
@@ -0,0 +1,182 @@
+    name="${1}"
+  elif [ "X_${1}_X" = "X_current_X" ] ; then
+    name="${1}"
+  else
+    name="v${1}"
+    version="-${1}-1"
+  fi
+fi
+echo "-------------------------------------------------------------------------------------------------------------------"
+echo "Installing Repose ${name} package"
+echo "-------------------------------------------------------------------------------------------------------------------"
+
+if [ "X_${name}_X" = "X_local_X" ] ; then
+  yum --nogpgcheck localinstall -y /vagrant/repose-valve-*.noarch.rpm /vagrant/repose-filter-bundle-*.noarch.rpm /vagrant/repose-extensions-filter-bundle-*.noarch.rpm /vagrant/repose-experimental-filter-bundle-*.noarch.rpm
+else
+  yum install -y repose-valve${version} repose-filter-bundle${version} repose-extensions-filter-bundle${version} repose-experimental-filter-bundle${version}
+fi
+mkdir -p /etc/systemd/system/repose-valve.service.d && echo "[Service]\nEnvironment=\"JAVA_OPTS=-Xdebug -Xrunjdwp:transport=dt_socket,address=10037,server=y,suspend=n\"" > /etc/systemd/system/repose-valve.service.d/local.conf
+sed -i '/JAVA_OPTS/c\JAVA_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,address=10037,server=y,suspend=n"' /etc/sysconfig/repose
+SCRIPT
+
+$copy_configs = <<SCRIPT
+echo "-------------------------------------------------------------------------------------------------------------------"
+echo "Copying configs"
+echo "-------------------------------------------------------------------------------------------------------------------"
+
+chmod a+rwx /vagrant
+if [[ -d "/vagrant/etc_repose" ]] ; then
+    cp --force /vagrant/etc_repose/* /etc/repose/
+fi
+SCRIPT
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "centos/7"
+
+  # Expose the Repose Java Debug Wire Protocol (JDWP) port
+  config.vm.network :forwarded_port, guest: 10037, host: 18039
+  #config.vm.network "private_network", ip: "192.168.33.39"
+  # Expose the Repose port
+  config.vm.network :forwarded_port, guest: 8080, host: 18089
+
+  config.vm.provision "shell", inline: $update_yum
+  # IF the Repose version is < 8.1.0.0,
+  # THEN the old Daemonize needs to be manually installed since it is no longer packaged.
+  # TODO: Make this conditional.
+  config.vm.provision "shell", inline: $install_old_repose_dependencies
+  config.vm.provision "shell", inline: $install_repose_dependencies
+  #config.vm.provision "shell", inline: $other_tools
```

Comment:
Same. This is now intended to be an automated environment, so they aren't needed.